### PR TITLE
Added createdAt and updatedAt fields to Channel

### PR DESCRIPTION
**Pull request by dima-ko**

### CODE REVIEW

**Review by welsinga — APPROVED** [review]
Using a constant is much nicer next time

**Comment by dzolnai** [comment]
Shouldn't these work with Date types instead of String?

File: android-client/src/main/java/io/msgs/v2/entity/Channel.java [hunk 1]
```diff
@@ -69,8 +95,17 @@ public Channel setCreatedAt(String createdAt) {
...
-        return _getString("updatedAt");
+    public Date getUpdatedAt() {
+        String createdAtString = _getString("updatedAt");
+        if (createdAtString == null) {
+            return null;
+        }
+        try {
+            return DATE_FORMAT.parse(createdAtString);
+        } catch (ParseException e) {
+            Log.e(TAG, "Can't parse 'updatedAt' date: " + createdAtString);
```
Comment (welsinga) [inline android-client/src/main/java/io/msgs/v2/entity/Channel.java:106]: Aren't you using Timber?

**Review by welsinga — CHANGES_REQUESTED** [review]

File: android-client/src/main/java/io/msgs/v2/entity/Channel.java (hunk 1 above)
Comment (dzolnai) [reply android-client/src/main/java/io/msgs/v2/entity/Channel.java:106]: No external library depedencies

**Review by welsinga — APPROVED** [review]

**Comment by dzolnai** [comment]
Closing in favor of #5

**Pull request closed without merging**